### PR TITLE
Update new-field.md

### LINKED ISSUE #22662: Warning in docs no longer accurate - SQLite provider now supports column change migrations without manual effort
# Issue description

Docs include this warning

image

However, I was able to successfully generate and apply a migration using

```powershell
dotnet-ef migrations add rating
dotnet-ef database update
```

Docs claim that migrations must be done manually when editing columns.

> The workaround for the SQLite limitations is to manually write migrations code to perform a table rebuild when something in the table changes. 

However, my migration seems to have done it for me?

```powershell
PS D:\Repos\Other\emotify\RazorPagesMovie> dotnet-ef migrations add rating
Build started...
Build succeeded.
An operation was scaffolded that may result in the loss of data. Please review the migration for accuracy.
Done. To undo this action, use 'ef migrations remove'
PS D:\Repos\Other\emotify\RazorPagesMovie> dotnet-ef database update
Build started...
Build succeeded.
info: Microsoft.EntityFrameworkCore.Database.Command[20101]
      Executed DbCommand (7ms) [Parameters=[], CommandType='Text', CommandTimeout='30']
      SELECT COUNT(*) FROM "sqlite_master" WHERE "name" = '__EFMigrationsHistory' AND "type" = 'table';
info: Microsoft.EntityFrameworkCore.Database.Command[20101]
      Executed DbCommand (1ms) [Parameters=[], CommandType='Text', CommandTimeout='30']
      SELECT COUNT(*) FROM "sqlite_master" WHERE "name" = '__EFMigrationsHistory' AND "type" = 'table';
info: Microsoft.EntityFrameworkCore.Database.Command[20101]
      Executed DbCommand (1ms) [Parameters=[], CommandType='Text', CommandTimeout='30']
      SELECT "MigrationId", "ProductVersion"
      FROM "__EFMigrationsHistory"
      ORDER BY "MigrationId";
info: Microsoft.EntityFrameworkCore.Database.Command[20101]
      Executed DbCommand (0ms) [Parameters=[], CommandType='Text', CommandTimeout='30']
      ALTER TABLE "Movie" ADD "Rating" TEXT NULL;
info: Microsoft.EntityFrameworkCore.Database.Command[20101]
      Executed DbCommand (0ms) [Parameters=[], CommandType='Text', CommandTimeout='30']
      CREATE TABLE "ef_temp_Movie" (
          "ID" INTEGER NOT NULL CONSTRAINT "PK_Movie" PRIMARY KEY AUTOINCREMENT,
          "Genre" TEXT NULL,
          "Price" decimal(18,2) NOT NULL,
          "Rating" TEXT NULL,
          "ReleaseDate" TEXT NOT NULL,
          "Title" TEXT NULL
      );
info: Microsoft.EntityFrameworkCore.Database.Command[20101]
      Executed DbCommand (0ms) [Parameters=[], CommandType='Text', CommandTimeout='30']
      INSERT INTO "ef_temp_Movie" ("ID", "Genre", "Price", "Rating", "ReleaseDate", "Title")
      SELECT "ID", "Genre", "Price", "Rating", "ReleaseDate", "Title"
      FROM "Movie";
info: Microsoft.EntityFrameworkCore.Database.Command[20101]
      Executed DbCommand (0ms) [Parameters=[], CommandType='Text', CommandTimeout='30']
      PRAGMA foreign_keys = 0;
info: Microsoft.EntityFrameworkCore.Database.Command[20101]
      Executed DbCommand (0ms) [Parameters=[], CommandType='Text', CommandTimeout='30']
      DROP TABLE "Movie";
info: Microsoft.EntityFrameworkCore.Database.Command[20101]
      Executed DbCommand (0ms) [Parameters=[], CommandType='Text', CommandTimeout='30']
      ALTER TABLE "ef_temp_Movie" RENAME TO "Movie";
info: Microsoft.EntityFrameworkCore.Database.Command[20101]
      Executed DbCommand (0ms) [Parameters=[], CommandType='Text', CommandTimeout='30']
      PRAGMA foreign_keys = 1;
info: Microsoft.EntityFrameworkCore.Database.Command[20101]
      Executed DbCommand (0ms) [Parameters=[], CommandType='Text', CommandTimeout='30']
      INSERT INTO "__EFMigrationsHistory" ("MigrationId", "ProductVersion")
      VALUES ('20210706014402_rating', '5.0.7');
Done.
```

The warning is no longer entirely true and should probably be updated.

# Software versions

Check the .NET target framework(s) being used, and include the version number(s).

* [x] .NET Core
* [x] .NET 5.0 or later
* [ ] .NET Framework - unsure??
* [ ] .NET Standard - unsure??

<details>
<summary><strong>dotnet --info output</strong> or <strong>About VS info</strong></summary>

```console
PS D:\Repos\Other\emotify\RazorPagesMovie> dotnet --info
.NET SDK (reflecting any global.json):
 Version:   5.0.301
 Commit:    ef17233f86

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19042
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\5.0.301\

Host (useful for support):
  Version: 5.0.7
  Commit:  556582d964

.NET SDKs installed:
  5.0.301 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 5.0.7 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 5.0.7 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 5.0.7 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

To install additional .NET runtimes or SDKs:
  https://aka.ms/dotnet-download
```
</details>

#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 01013ef6-c55c-ff58-571c-ddc4435993af
* Version Independent ID: 24699fe5-9744-1796-0798-f7b211ec0f2f
* Content: [Part 7, add a new field](https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/new-field?view=aspnetcore-5.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/razor-pages/new-field.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/razor-pages/new-field.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #22662

@TeamDman please review

Internal review URLs
* [Add a rating](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/new-field?view=aspnetcore-5.0&branch=pr-en-us-22682&tabs=visual-studio-code#add-a-migration-for-rating)

### CODE REVIEW

File: aspnetcore/tutorials/razor-pages/new-field.md [hunk 1]
```diff
@@ -112,24 +112,43 @@ Another option is to delete the database and use migrations to re-create the dat
 
 # [Visual Studio Code / Visual Studio for Mac](#tab/visual-studio-code+visual-studio-mac)
 
-### Drop and re-create the database
+Use the following commands to add a migration for the rating field:
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/razor-pages/new-field.md:115]:
```suggestion
### Add a migration for rating
Use the following commands to add a migration for the rating field:
```

File: aspnetcore/tutorials/razor-pages/new-field.md [hunk 2]
```diff
@@ -112,24 +112,44 @@ Another option is to delete the database and use migrations to re-create the dat
...
+dotnet-ef migrations add rating
+dotnet-ef database update
+```
+
+The `dotnet-ef migrations add rating` command tells the framework to:
+
+* Compare the `Movie` model with the `Movie` database schema.
+* Create code to migrate the database schema to the new model.
+
+The name `Rating` is arbitrary and is used to name the migration file. It's helpful to use a meaningful name for the migration file.
```
Comment (TeamDman) [inline aspnetcore/tutorials/razor-pages/new-field.md:128]:
```suggestion
The name `rating` is arbitrary and is used to name the migration file. It's helpful to use a meaningful name for the migration file.
```

File: aspnetcore/tutorials/razor-pages/new-field.md [hunk 3]
```diff
@@ -112,24 +112,44 @@ Another option is to delete the database and use migrations to re-create the dat
...
+
+The `database update` command tells the framework to apply the schema changes to the database and to preserve existing data.
```
Comment (TeamDman) [inline aspnetcore/tutorials/razor-pages/new-field.md:130]:
```suggestion
The `dotnet-ef database update` command tells the framework to apply the schema changes to the database and to preserve existing data.
```

**Review by TeamDman — APPROVED** [review]
LGTM, some nitpicks but overall great!

**Comment by Rick-Anderson** [comment]
@TeamDman Those are great changes, sorry I missed them. Can you create a new issue and fix them? The changes will be live in 5 minutes.

I did all my changes in the browser. Let me know if you have any questions.

**Comment by TeamDman** [comment]
Sure thing!

**Pull request merged**